### PR TITLE
Added links to the OpenShift Image Streams and Templates Library

### PR DESCRIPTION
**Pull request by bmcelvee**
https://trello.com/c/VMTxZmXu/356-document-better-distribution-of-templates-examples-evg

### CODE REVIEW

**Comment by bmcelvee** [comment]
Previews: 
http://file.rdu.redhat.com/~bmcelvee/10042017/document_templates_examples/architecture/core_concepts/builds_and_image_streams.html#image-streams
http://file.rdu.redhat.com/~bmcelvee/10042017/document_templates_examples/dev_guide/templates.html#overview

**Comment by bmcelvee** [comment]
@coreydaley PTAL 😸

**Comment by coreydaley** [comment]
@bmcelvee lgtm

**Comment by bmcelvee** [comment]
@ahardin-rh peer review please

File: architecture/core_concepts/builds_and_image_streams.adoc [hunk 1]
```diff
@@ -170,8 +170,9 @@ endif::[]
 == Image Streams
 
 An _image stream_ comprises any number of
-xref:containers_and_images.adoc#docker-images[Docker-formatted container images] identified by tags.
-It presents a single virtual view of related images, similar to an image repository, and may contain images from any of the following:
+xref:containers_and_images.adoc#docker-images[Docker-formatted container images]
+identified by tags. It presents a single virtual view of related images, similar
```
Comment (ahardin-rh) [inline architecture/core_concepts/builds_and_image_streams.adoc]: Need to update this path:
xref:../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted container images]

**Comment by ahardin-rh** [comment]
@bmcelvee Just one comment from me. Otherwise, looks good! :boom:

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../architecture/core_concepts/builds_and_image_streams.adoc#architecture-core-concepts-builds-and-image-streams[Builds and Image Streams]
|Added link to OpenShift Image Streams and Templates library in the xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[Image Streams] section.
%
|xref:../dev_guide/templates.adoc#dev-guide-templates[Templates]
|Added link to OpenShift Image Streams and Templates library in the xref:../dev_guide/templates.adoc#overview[Overview] section.
%

**Pull request merged**